test/sql/types/nested/list/test_list_reverse.test

```diff
@@ -0,0 +1,45 @@
+# name: test/sql/types/nested/list/test_list_reverse.test
+# description: Test reversing the order of the elements of a list
+# group: [list]
```

Comment:
Can we add some coverage tests with different vector types?
You can use the `test_vector_types` function to help with this, see `test/sql/types/nested/map/test_map_subscript_vector_types.test` for an example.

Also can you try with a column as input instead of a constant expression?
(using something bigger than 2048 here just to also test how this behaves with lists bigger than STANDARD_VECTOR_SIZE)
```sql
statement ok
create table tbl as select range(5000) as list;

query I nosort reversed_list
select list_sort(range(5000), 'desc');

query I nosort reversed_list
select list_reverse(list) from tbl;
```